.travis.yml

```diff
@@ -0,0 +1,20 @@
+---
+language: node_js
```

Comment:
@igillis I assume this is an ember-cli default for working with Travis? Can we take it out for now, but hopefully someday add it back ... and get Travis ... ?